graphene_federation/tests/test_scalar.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any
+
+from graphene import Scalar, String, ObjectType
+
+from graphene_federation import build_schema
+
+
+def test_custom_scalar():
+    class AddressScalar(Scalar):
+        base = String
+
+        @staticmethod
+        def coerce_address(value: Any):
+            ...
+
+        serialize = coerce_address
+        parse_value = coerce_address
+
+        @staticmethod
+        def parse_literal(ast):
+            ...
+
+    class Query(ObjectType):
+        test = String(x=AddressScalar())
+
+    build_schema(query=Query, enable_federation_2=True)
```

Comment:
should we add an assert here?

Reply:
@patrick91, I will around this! Issue #13 could not build schema; will add an assertion on the schema!

Reply:
@patrick91 - Test case modified.